packages/docs/operations-manual/validator-explorer.md

```diff
@@ -0,0 +1,83 @@
+
+Assuming you have already deployed your Validator Group via a `ReleaseGold` contract, you will need these environment variables set to claim your domain.
+
+### Environment variables
+
+| Variable                             | Explanation                                                                                                                          |
+| ------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------ |
+| CELO_VALIDATOR_GROUP_RG_ADDRESS         | The `ReleaseGold` contract address for the Validator Group                                                                                          |
+| CELO_VALIDATOR_RG_ADDRESS         | The `ReleaseGold` contract address for the Validator                                                                                                 |
+| CELO_VALIDATOR_SIGNER_ADDRESS        | The address of the validator signer authorized by the validator account                                                              |
+| CELO_VALIDATOR_GROUP_SIGNER_ADDRESS  | The address of the validator (group) signer authorized by the validator account
+
+First let's create the metadata file:
+
+```
+# On your local machine
+celocli account:create-metadata ./group_metadata.json --from $CELO_VALIDATOR_GROUP_RG_ADDRESS
+```
+
+Now we can set the group's name:
+
+```
+# On your local machine
+celocli releasegold:set-account --contract $CELO_VALIDATOR_GROUP_RG_ADDRESS --property name --value Example.com
+```
+
+Now we can generate a claim for the domain associated with this name `example.com`:
+
+```
+# On your local machine
+celocli account:claim-domain ./group_metadata.json --domain example.com --from $CELO_VALIDATOR_GROUP_SIGNER_ADDRESS
+```
+
+This will output your claim signed under the provided signer address. This output should then be recorded via a `TXT Record` on your desired domain, so in this case we should add a `TXT Record` to `example.com` with this signed output.
+
+Once that record is added, we can then register this metadata under on our `Validator Group` account for external validation.
+
+Before we do this, you may also want to associate some validators with this domain. In order to do so, you will need to claim each validator address on your group's metadata. You will also need to claim your group account on each of your validator's metadata to complete the association. We will run through an example of a single validator now:
```

Comment:
Maybe elaborate a bit more why one would want to do that? Arguably its most important for the group, but it makes sense to claim all the validators as well to convey that identity